lib/msal-common/src/authority/TrustedAuthority.ts

```diff
@@ -0,0 +1,84 @@
+import { Constants } from "../utils/Constants";
+import { INetworkModule } from "../network/INetworkModule";
+import { OpenIdConfigResponse } from "./OpenIdConfigResponse";
+import { InstanceMetadataType } from "./InstanceMetadataType";
+import { IInstanceDiscoveryMetadata } from "./IInstanceDiscoveryMetadata";
+import { ClientConfigurationError } from "../error/ClientConfigurationError";
+
+export class TrustedAuthority {
+    private static InstanceMetadata: InstanceMetadataType = {};
```

Comment:
Is there any reason to make these static? I would rather only make functions and fields static if needed.

Reply:
Things like knownAuthorities and the network interface can then be passed into the constructor and re-used

Reply:
I received feedback when I implemented knownAuthorities originally that it should be static. The idea is that it's a set once variable to prevent trying to switch up your knownAuthorities mid-stream so we don't want it to be tied to an instance where we may need to rebuild it.